BBHX_Phenom.py

```
@@ -229,8 +229,8 @@ def _bbhx_fd(
             t0=0
         )
     else:
-        err_msg = f"Don't recognise reference frame {ref_frame}. "
-        err_msg = f"Known frames are 'LISA' and 'SSB'."
+        err_msg = f"Don't recognise reference frame {ref_frame}. Known frames are 'LISA' and 'SSB'."
+        warn(err_msg, RuntimeWarning)
```

I think this should raise a `ValueError` rather than a warning, since the code should not return if the `ref_frame` is an invalid value.